ui/Screen/DesignSystemGuide.svelte

```diff
@@ -356,79 +341,88 @@
 
 <ModalLayout full>
   <div class="layout">
-    <Title variant="huge" style="margin-bottom: 92px">Primitives</Title>
-
-    <Section title="Colors" subTitle="Primary, secondary and grays">
-
-      {#each colors as color}
-        <Text
-          style="background-color: var({color}); margin-bottom: 8px;
-          border-radius: 2px; padding: 4px 8px;">
-          {color}
-        </Text>
-      {/each}
-    </Section>
+    <h1 style="margin-bottom: 92px">Primitives</h1>
 
     <Section
-      title="Typography"
+      title="Typography updated"
```

Comment:
```suggestion
      title="Typography"
```

Once the PR is merged this will be the de facto typography.

Reply:
improve design system layout for typography

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/749/commits/f82d29b13a9575db8c724bf43b4772f4ed6f2a6d